### DISCUSSION TITLE
Preview function for Wagtail stream field form

### DISCUSSION THREAD

**Discussion by wes-otf** [OpenTechFund/opentech.fund#4036] [General]
As the Program team has built out forms quite a bit the last few months something that keeps coming up is an inability to preview forms like Reviewer Forms, Application forms (without creating a fund/lab), etc.

So far I've been having to copy forms from the test DB to the live DB for staff that wanted to see the form before publishing it. It would be a nice addition if these things were able to be previewed similar to the preview that happens when building a Fund/Lab in Wagtail. Is this a possibility at all/been discussed before?

**Comment by frjo** [comment]
I do not think it has come up before. If staff finds it useful we should add it.

**Comment by wes-otf** [comment]
Yeah this would be sweet! Like I said I think there have been 2 or 3 forms this month I've had to move between the databases so staff could preview so this would definitely be used.
